dot/network/service_test.go

```diff
@@ -84,11 +86,13 @@ func createTestService(t *testing.T, cfg *Config) (srvc *Service) {
 
 	if cfg.TransactionHandler == nil {
 		mocktxhandler := &MockTransactionHandler{}
-		mocktxhandler.On("HandleTransactionMessage", mock.AnythingOfType("*TransactionMessage")).Return(nil)
+		mocktxhandler.On("HandleTransactionMessage", mock.AnythingOfType("*network.TransactionMessage")).Return(true, nil)
 		mocktxhandler.On("TransactionsCount").Return(0)
 		cfg.TransactionHandler = mocktxhandler
 	}
 
+	cfg.SlotDuration = time.Second
```

Comment:
Dumb question: can we have this smaller to have faster tests?

Reply:
Reduced it to `100ms`

Reply:
yet another dumb question, is there a reason we can't have it to 1ns?

Reply:
No don't worry about it. I was curious if we could set it to an arbitrary value and have it event driver rather than time driven. Since we already have so many tests using sleeps and all, it's fine to have one more I guess